caiman/tests/test_memmapping.py

```diff
@@ -0,0 +1,19 @@
+from caiman import mmapping
```

Comment:
I don't understand this test. Please explain.

Reply:
I believe it should be clearer after my latest commit which fixes the merge artifact.

It's a simple unit test designed to make sure that `load_memmap` raises the proper error if given a non-`.mmap` file.